docs/modules/ROOT/partials/proc-persistence-kafkasql-tls.adoc

```diff
@@ -53,14 +53,23 @@ spec:
     userOperator: {}
 ----
 
-. Click the *Kafka Topic* tab, and then *Create Kafka Topic* to create the required `global-id-topic`:
+. The default Kafka topic name that {registry} uses to store data is `kafkasql-journal`.
```

Comment:
This is not a step. Suggest removing `.` and adding `+` on previous line. 

Suggest removing `and` + `.` from end of bullets, i.e., use only:
** `REGISTRY_KAFKASQL_TOPIC_AUTO_CREATE=true` 
** `REGISTRY_KAFKASQL_TOPIC=kafkasql-journal`